bridges/snowbridge/runtime/test-common/src/lib.rs

```diff
@@ -193,9 +195,104 @@ pub fn send_transfer_token_message_success<Runtime, XcmConfig>(
 
 			let digest = included_head.digest();
 
-			//let digest = frame_system::Pallet::<Runtime>::digest();
 			let digest_items = digest.logs();
 			assert!(digest_items.len() == 1 && digest_items[0].as_other().is_some());
+
+			let digest_item = &digest_items[0];
+			match digest_item {
+				DigestItem::PreRuntime(_, _) => panic!("is PreRuntime"),
+				DigestItem::Consensus(_, _) => panic!("is Consensus"),
+				DigestItem::Seal(_, _) => panic!("is Seal"),
+				DigestItem::Other(veccc) => {
+					assert_eq!(veccc[0], 0);
```

Comment:
I do not understand this assert, why assert only the first element?

Reply:
Whoops, good catch. I tested something locally and forgot to revert this change. Reverted here: https://github.com/Snowfork/polkadot-sdk/pull/112/commits/b570e54a412f27ff4e157408ac365d17e5c91e90